pkg/arvo/sys/vane/ames.hoon

```diff
@@ -3061,14 +3061,20 @@
   (en:crub:crypto symmetric-key (jam plaintext))
 ::  +decrypt: decrypt packet content to a $shut-packet or die
 ::
+::    TODO: alter |acru interface
+::
 ++  decrypt
   |=  [=symmetric-key ciphertext=@]
   ^-  shut-packet
-  ::
-  ;;  shut-packet
-  %-  cue
-  %-  need
-  (de:crub:crypto symmetric-key ciphertext)
+  =+  ;;([iv=@ len=@ cyf=@] (cue ciphertext))
+  ::  assert cyphertext length <= claimed length <= max bytes
+  ::
+  ?.  (lte (met 3 cyf) len)
+    ~|(ames-len-small+len^(met 3 cyf) !!)
+  ?.  (lte len 1.024)
+    ~|(ames-len-large+len !!)
```

Comment:
I think this would be clearer in `bloq=13`, to match the other packet-length operations in %ames.

Reply:
I could do
```
?.  (lte len ^~((bex (sub 13 3))))
```
so that someone could change the packet length by string-replacing `13` with some other number.  Let me know if you know a better way to express this.

Reply:
I don't think you need to check `len` at all here, I'd just `?>  (lte (met 13 cyf) 1)` or even `?>  ?=(?(%0 %1) (met 13 cyf))`.

Reply:
The decryption call validates the relationship between `len` and `cyf`.